src/jit/gentree.cpp

```diff
@@ -6308,6 +6308,10 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
```

Comment:
Here is the issue tracking the above two suggestions:
https://github.com/dotnet/coreclr/issues/3246